runtime/src/init.rs

```diff
@@ -26,8 +27,11 @@ pub fn start_runtime(initializer: Box<dyn Initializer>, config: Config) {
     // Initialize runtime identity with runtime attestation key and runtime encryption key.
     let identity = Arc::new(Identity::new());
 
+    // Initialize the async Tokio runtime.
+    let tokio_runtime = new_tokio_runtime();
```

Comment:
Maybe cloning outside of the constructor, like identity is.
```suggestion
    let tokio_runtime = new_tokio_runtime().handle();

    // Initialize the dispatcher.
    let dispatcher = Dispatcher::new(tokio_runtime.clone(), initializer, identity.clone());
```